istioctl/pkg/install/k8sversion/version_test.go

```diff
@@ -76,21 +76,15 @@ func TestExtractKubernetesVersion(t *testing.T) {
 		errMsg   error
 		isValid  bool
 	}{
-		{
```

Comment:
I'm not sure why we removed the 1.16 here as well as in the var above, as it's just a check of the extractKubernetesVersion(). We already test similar 1.1.x versions so it may be redundant.

Reply:
This is just to keep test cases limited to what we support otherwise this will keep increasing with every release. I'm fine with keeping this but it will unnecessarily add more burden.